content/tutorials/custom-zk-chain/20.customizing-your-chain.md

```diff
@@ -244,102 +260,105 @@ Now that the chain is initialized, you can start the chain server:
 zkstack server
 ```
 
+Your new chain should be running at port `3150`.
+You can confirm this by looking for a log in the server that says `Initialized HTTP API on 0.0.0.0:3150`,
+or by looking in your `my_elastic_network/chains/custom_zk_chain/configs/general.yaml` file at the `web3_json_rpc:http_port` value.
+
 ## Bridging the base token to your chain
 
 The last step is to bridge some ERC20 tokens from the L1 to the new chain.
 Base tokens can not be minted on the L2 without being backed by the corresponding L1 amount. In the future, there will be more flexibility for this.
 
-Open a new terminal and use ZKsync CLI to bridge the tokens with the command below:
+Instead of using the ZKsync CLI here, we'll use the `zksync-ethers` SDK.
+To try this out, open your hardhat project `zk-chain-test` from the previous section,
+and run the commands below to create a new script file:
 
 ```bash
-npx zksync-cli bridge deposit --token <0x_YOUR_TOKEN_ADDRESS> \
---rpc=http://localhost:3050 \
---l1-rpc=http://localhost:8545
+mkdir scripts
+touch scripts/depositBaseToken.ts
 ```
 
-```shell
-? Amount to deposit 5
-? Private key of the sender 0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110
-? Recipient address on L2 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049
+Then copy and paste the script below.
+
+```ts [depositBaseToken.ts]
+:code-import{filePath="custom-zk-chain/scripts/depositBaseToken.ts"}
 ```
 
-To verify that this worked, let's check the new balance of our address on the L2 chain:
+Before running the script:
 
-```bash
-npx zksync-cli wallet balance \
---address 0x36615Cf349d7F6344891B1e7CA7C72883F5dc049 \
---rpc http://localhost:3050
+1. Update the RPC url in the `hardhat.config.ts` file so it connects to port `3150` instead of `3050`.
+1. Update the `WALLET_PRIVATE_KEY` variable in your `.env` file to your governor address private key,
```

Comment:
```suggestion
1. Update the RPC url in the `hardhat.config.ts` file so it connects to port `3150` instead of `3050`.
2. Update the `WALLET_PRIVATE_KEY` variable in your `.env` file to your governor address private key,
```

Reply:
this is handled automatically by the nuxt template. so it shows up as 2 on the page